src/TheResumeTheme/Recipes/TheResumeTheme.recipe.json

```diff
@@ -1292,8 +1292,8 @@
       "name": "media",
       "Files": [
         {
-          "Path": "img/profile.jpg",
-          "Base64": "[file:base64('img/profile.jpg')]"
+          "SourcePath": "img/profile.jpg",
+          "TargetPath": "img/profile.jpg"
```

Comment:
Why are these changes necessary?

Reply:
The profile image on the home page is missing without it causing the UI test to fail. I didn't find what change in OC broke it, but we don't have any other cases of using an image like this in Lombiq code (according to GH code search).
This should be contributed back to the original repo too.

Reply:
OK, please do once we're done.